src/main/kotlin/no/nav/syfo/behandlendeenhet/EnhetService.kt

```diff
@@ -26,22 +28,27 @@ constructor(
         callId: String,
         personIdentNumber: PersonIdentNumber
     ): BehandlendeEnhet? {
-        val geografiskTilknytning = pdlConsumer.geografiskTilknytning(personIdentNumber)
-        val isEgenAnsatt = skjermedePersonerPipConsumer.erSkjermet(callId, personIdentNumber.value)
-
-        val graderingList = pdlConsumer.person(personIdentNumber)?.gradering()
-
-        val behandlendeEnhet = norgConsumer.getArbeidsfordelingEnhet(
-            callId,
-            graderingList?.toArbeidsfordelingCriteriaDiskresjonskode(),
-            geografiskTilknytning,
-            isEgenAnsatt
-        ) ?: return null
-
-        return if (isEnhetUtvandret(behandlendeEnhet)) {
-            getEnhetNAVUtland(behandlendeEnhet)
-        } else {
-            behandlendeEnhet
+        try {
+            val geografiskTilknytning = pdlConsumer.geografiskTilknytning(personIdentNumber)
+            val isEgenAnsatt = skjermedePersonerPipConsumer.erSkjermet(callId, personIdentNumber.value)
+
+            val graderingList = pdlConsumer.person(personIdentNumber)?.gradering()
+
+            val behandlendeEnhet = norgConsumer.getArbeidsfordelingEnhet(
+                callId,
+                graderingList?.toArbeidsfordelingCriteriaDiskresjonskode(),
+                geografiskTilknytning,
+                isEgenAnsatt
+            ) ?: return null
+
+            return if (isEnhetUtvandret(behandlendeEnhet)) {
+                getEnhetNAVUtland(behandlendeEnhet)
+            } else {
+                behandlendeEnhet
+            }
+        } catch (e: PdlRequestFailedException) {
+            LOG.warn(e.toString())
+            return null
         }
```

Comment:
Ref: https://trello.com/c/lseYzuHW/615-syfobehandlendeenhet-fjerne-un%C3%B8dvendig-logging

Det som står i syfodok er utdatert, vi bruker PDL nå og ikke TPS. Dvs. at vi kanskje kan gjøre noe med disse "feilene".
Det at vi ikke finner geografisk tilknytning til en person er altså noe vi potensielt kan gjøre noe med om vi finner ut av hvilke identer det gjleder(e.g. med secured logging) og sender disse til MFN/PDL-gjengen.

Derfor synes jeg ikke vi burde maskere disse og håpe på det beste, men heller prøve å finne ut av hva vi kan gjøre for å løse opp i det så vi finner GT på flest mulig personer.